static/app/components/replays/videoReplayerWithInteractions.tsx

```diff
@@ -117,16 +116,19 @@ export class VideoReplayerWithInteractions {
               ],
               id: 0,
             },
+            initialOffset: {
+              top: 0,
+              left: 0,
+            },
```

Comment:
`initialOffset` and the `id` fields were added so the types would conform properly. 

Before, with `data.node.id: 0` it wasn't inferring that this is a Document type, so it didn't care if initialOffset was present. Now it's gotta be included.